recipes/vtu11/all/conandata.yml

```
@@ -0,0 +1,5 @@
+sources:
+  "1.2.0":
```

Should match the versioning used by upstream https://github.com/phmkopp/vtu11/releases/tag/v1.2

```suggestion
  "1.2":
```